### PR TITLE
docs: update wrangler install

### PR DESCRIPTION
**Pull request by octref**
`wrangler` is recommended: https://developers.cloudflare.com/workers/wrangler/install-and-update/

```
Code > npm install --global @cloudflare/wrangler
npm WARN deprecated @cloudflare/wrangler@1.21.0: This package is for Wrangler v1.x and is no longer supported. Please visit https://www.npmjs.com/package/wrangler for Wrangler v2+.

added 22 packages in 1s

3 packages are looking for funding
  run `npm fund` for details
 Code > wrangler login
⛔  DEPRECATED ⛔
```

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -8,8 +8,8 @@ jobs:
     runs-on: ubuntu-latest
     name: Deploy
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v2
```
Comment (gr2m) [inline .github/workflows/deploy.yml:11]: Can you revert this change please?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -1,17 +0,0 @@
...
-    name: release
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
-        with:
-          node-version: lts/*
-      - run: npx semantic-release
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (gr2m) [inline .github/workflows/release.yml:17]: Revert this change please.

File: index.html [hunk 3]
```diff
@@ -35,7 +35,7 @@ <h1>
       <small>A Cloudflare Worker + GitHub Pages Login Example</small>
     </h1>
     <p id="signed-out">
-      <a href="https://github-oauth-login.gr2m.workers.dev"
+      <a href="https://github-oauth-login.octref.workers.dev"
```
Comment (gr2m) [inline index.html:38]: revert this change please

File: index.html [hunk 4]
```diff
@@ -47,7 +47,7 @@ <h1>
     </p>
 
     <script>
-      const WORKER_URL = "https://github-oauth-login.gr2m.workers.dev";
+      const WORKER_URL = "https://github-oauth-login.octref.workers.dev";
```
Comment (gr2m) [inline index.html:50]: revert this change please

**Review by gr2m — CHANGES_REQUESTED** [review]

**Comment by octref** [comment]
oops, i'm sorry. i totally forgot that i sent a PR and kept doing work on it. i'll send a new PR.
